content/en/pod.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Pod
+status: Feedback Appreciated
+category: concept
+tags: ["infrastructure", "fundamental", ""]
+---
+
+## What it is
+
+In [cloud native computing](/cloud-computing/), a "pod" is a basic building block for deploying and managing containerized applications. 
+A pod represents a single instance of an application, and can contain one or more [containers](/container/).
+[Kubernetes](/kubernetes/) manages pods as part of a larger deployment, and can scale pods up or down depending on the application's resource requirements.
+
+## Problem it addresses
+
+Containers provide a lightweight and efficient way to package and deploy applications, but they can't do everything on their own. 
+Applications often require additional resources and services, such as storage or networking, which need to be managed and configured separately.
+
+This is where pods come in. A pod provides a way to group containers together and manage them as a single unit. 
+This makes it easier to ensure that all the resources and services in application needs are available and configured correctly.
+
+## How it helps
+
+Isolation: Each pod has its own isolated environment, including its own network stack and file system. This ensures that containers in the same pod can't interfere with each other, and helps maintain security and stability.
+
+Resource sharing: Containers in the same pod can share resources, such as CPU and memory, which allows for efficient use of resources.
```

Comment:
As far as I know, code in containers is oblivious of other code it shares resources with (similar to VMs). What I assume can happen here is that operators set resource limits per pod that are then shared among all containers (?)

Reply:
I think you're correct @CathPag, containers in a pod share resources, but each container within a pod can have their own limits